tests/conjectures_tests.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+from ramanujan.LHSHashTable import LHSHashTable
+from ramanujan.enumerators.EfficientGCFEnumerator import EfficientGCFEnumerator
+from ramanujan.poly_domains.CartesianProductPolyDomain import CartesianProductPolyDomain
+from ramanujan.poly_domains.Zeta3Domain1 import Zeta3Domain1
+from ramanujan.constants import g_const_dict
+
+
+def get_testable_data(refined_res_list):
+    '''
+        On those test we have no use for the bloom key or lhs dict pointers
+        exctact the meaning full data for testing
+    '''
+    data = []
+    for i in refined_res_list:
+        data.append((
+            i.rhs_an_poly,
+            i.rhs_bn_poly,
+            i.c_top,
+            i.c_bot))
+    return data
```

Comment:
i don't really understand why we need this...
also - these are API tests, so if we check only partial data then we kind of miss the point..
These tests need to test our SW from the highest level possible - use level.
so basically - check that all the results we expect exist

Reply:
I've wanted to use this functionality:
self.assertIn(
            ((2, 1, 3, 1), (-1,), (8, 0), (0, 7)), 
            results)

when the results given from the enumerator is actually a RefinedResults object (named tuple). It contained some data we don't have to use if all we want to do is to compare the results

regarding the second point, those are the exact same tests for the old API. I haven't removed any test cases...
I have no solid opinion for either option, and checking all of the results shouldn't be too much of a hassle. Should I do it?

Reply:
i leave it up to you :) 
just bear in mind that this is a API test - should simulate the user's use case.